change/react-native-windows-init-880aa0d6-47c8-4010-afdd-66785b084790.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Comment:
"None" means we will not release a new version. Could you change this to "patch"?